.github/workflows/notify.yml

```diff
@@ -13,8 +13,7 @@ jobs:
   notify-teams:
     runs-on: ubuntu-latest
     environment:
-      environment: ${{ env.environment }}
-      timezone: ${{ env.timezone }}
+      name: dev
```

Comment:
environment:
      name: ${{ needs.setup.outputs.environment }}
      
Can we please follow similar pattern i.e. `sca.yml` 
![image](https://github.com/UK-Export-Finance/dtfs2/assets/32125108/ed31c68d-5040-41d0-8a69-deabb433422b)

Reply:
I originally had something similar to this setup but realised it was a redundant step that doesn't give us anything. Happy to add it back in and will now just wanted to reduce number of steps if it wasn't needed!

Reply:
Done :)